Add_User.py

```diff
@@ -0,0 +1,13 @@
+def Add_User():
+    file = open("accountfiles.txt","a")
+    #Calculates ID using the length of the file
+    with open(r"accountfiles.txt", 'r') as fp:
```

Comment:
Why this r before the "database.txt"? r is for relative path? (maybe it is ok to use it but I have never use it before) (I am just curious)

Reply:
Haha I actually found this bit of code online and haven't tried it without the r, I will try it